python/ray/dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -88,7 +88,7 @@ def max_plus_pending(max_resource, pending_resource):
     Panel(
         id=33,
         title="Scheduler Actor State",
-        description="Current number of actors in a particular state.\n\nState: the actor state, as described by rpc::ActorTableData proto in gcs.proto.",
+        description='Note: not impacted by "Instance" variable.\n\nCurrent number of actors in a particular state.\n\nState: the actor state, as described by rpc::ActorTableData proto in gcs.proto.',
```

Comment:
@alanwguo can you please file a ticket to fix it in Core and add a TODO here?

Reply:
looks like, this is already avialable with the `NodeAddress` label. Updated the graphs here